editor/src/clj/editor/ui.clj

```diff
@@ -2234,7 +2234,10 @@
     (do
       (.start (Thread. #(.browse desktop (as-url url))))
       true)
-    false))
+    (do
+      (log/warn :message (format "Cannot open browser"))
+      (if (os/is-linux?) (log/warn :message (format "Installing gvfs may fix this error")))
+    false)))
```

Comment:
We don't have to use `format` in this case, since we are not interpolating. We can also use `when` instead of `if`, since there is not `else` branch. We could also concatenate the strings to use a single `log/warn`. Additionally, we should fix the indentation of the returned bools. The end result would look something like this
```clojure
(do
  (log/warn :message (str "Cannot open browser." (when (os/is-linux?) "Installing gvfs may fix this.")))
  false)
```